package.json

```diff
@@ -770,7 +770,7 @@
     "gulp-typescript": "^6.0.0-alpha.1",
     "mocha": "^9.2.2",
     "moment": "^2.29.4",
-    "node-fetch": "^2.6.7",
+    "node-fetch": "^2.6.8",
```

Comment:
What's the need of increasing the version?

Reply:
I just ran npm install node-fetch and it took latest version